edgedb/protocol/protocol.pxd

```diff
@@ -94,6 +94,7 @@ cdef class SansIOProtocol:
 
         readonly bytes last_status
         readonly bytes last_details
+        readonly uint16_t minor_protocol_version
```

Comment:
it would be more convenient to work with the protocol version as a tuple, i.e. `(major, minor)`.

Reply:
Then it will be an `object` in cython, right? Which is subpar I think. We can make a property that returns tuple if we find some use case for that.

Reply:
It will be a `tuple`.  If you are worried about performance in a hot spot, we can always extract the components into integer components there.  The point is that doing version comparison checks as `self.version < (major, minor)` is more convenient and less error-prone.

Reply:
Okay, fixed.